pages/edge/tutorials/cdn.mdx

```diff
@@ -0,0 +1,223 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
+import StaticSite from "@components/deploy/quickstart/StaticSite";
+import edgeReactImg from "@assets/deploy/tutorials/edge-react.png";
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Using Wasmer Edge as a CDN
+
+In this tutorial, you will learn how to use Wasmer Edge as a CDN to host your static assets. We'll be using the [`static-web-server`](https://wasmer.io/wasmer/static-web-server)
+
+## Prerequisites
+
+The project requires the following tools to be installed on your system:
+
+- [Rust](https://www.rust-lang.org/tools/install)
+
+## Deploying a Static Site
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Initialize the project
+
+We'll initialize the project with the `static-site` template.
+
+Navigate to the directory where you want to create the project and run the following command:
+
+```bash
+$ wasmer app create
+```
+
+The above command will give you the following prompt:
+
+```bash
+App type:
+> Static website # Select Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+  Python Application
+Who should own this package?: <your-namespace>
+No package found or specified.
+What should the package be called? It will be published under wasmer: <your-app-name> # This will be the package name of your CDN
+What should be the name of the app? <NAME>.wasmer.app: <your-app-name> # This will be the name of your app
+Would you like to publish the app now? no # Select no for now
+Writing app config to '/path/to/your/project/app.yaml'
+To (re)deploy your app, run 'wasmer deploy'
+```
+
+Select the above configuration mentioned in the comments and press enter.
+
+The above command will initialize a blank project with the following structure.
+
+<FileTree>
+  <FileTree.Folder name="your-top-level-dir" defaultOpen>
+    <FileTree.Folder name="public" defaultOpen>
+      <FileTree.File name="index.html" />
+    </FileTree.Folder>
+    <FileTree.File name="app.yaml" />
+    <FileTree.File name="wasmer.toml" />
+  </FileTree.Folder>
+</FileTree>
+
+### Changing configuration for the CDN
+
+Let's first delete the `public` directory.
+
+```bash
+$ rm -r public
+```
```

Comment:
Why we want to start the guide by deleting a dir? Why do we need to delete it? It seems non-intuitive

Reply:
because we're adding the `config.toml` and the `public` dir inside `app`.
And when I did:
```toml
"."="."
```
It errors out that it's not able to find the root directory

Reply:
Changed it to "/app" = "."